torch/_decomp/__init__.py

```diff
@@ -40,9 +40,9 @@
 
 # TODO: relax key type here; torch registrations should be possible to; but
 # right now this type is accurate
-global_decomposition_table: Dict[
-    str, Dict[torch._ops.OperatorBase, Callable]
-] = defaultdict(dict)
+global_decomposition_table: Dict[str, Dict[torch._ops.OperatorBase, Callable]] = (
```

Comment:
was this file not linted properly before?

Reply:
I think i was using linter from fbcode but it is not good